apps/common/views.py

```diff
@@ -27,25 +31,34 @@ def post(self, request, *args, **kwargs):
 
         if order_status:
             order_id = event['data']['object']['metadata'].get('order_id')
+            user_id = event['data']['object']['metadata'].get('user_id')
+            payment_intent = event['data']['object'].get('id')
+
             Order.objects.filter(id=order_id).update(status=order_status)
 
+            if order_status == Order.Status.CONFIRMED:
+                Cart.objects.filter(user_id=user_id, is_archived=False).update(is_archived=True)
+
+                Invoice.objects.create(
```

Comment:
invoice-ul se creaza cand incepi plata, nu in webhook
se creeaza si se seteaza ca neachitat, si in webhook poti sa il setezi achitat